AUTHORS

```diff
@@ -49,6 +49,7 @@ Xiangyu Hu <xiangyu.hu at outlook.com>
 Xiaobing Jiang <s7v7nislands at gmail.com>
 Xiuming Chen <cc at cxm.cc>
 Zhenye Xie <xiezhenye at gmail.com>
+Ruben de Vries <ruben at rubensayshi.com>
```

Comment:
Please keep the name list in [alphabetical order](https://en.wikipedia.org/wiki/Alphabetical_order)